File: src/SkyBlock/enchants/bow/player/Paralyze.php
<?php


namespace SkyBlock\enchants\bow\player;


use pocketmine\entity\effect\EffectInstance;
use pocketmine\entity\effect\VanillaEffects;
use pocketmine\event\entity\EntityDamageByEntityEvent;
use pocketmine\player\Player;

class Paralyze extends BaseBowHitPlayerEnchant {

    static int $id = 117;

    /**
     * @param Player                    $attacker
     * @param Player                    $victim
     * @param int                       $enchantmentLevel
     * @param EntityDamageByEntityEvent $event
     */
    public function onActivation(Player $attacker, Player $victim, int $enchantmentLevel, EntityDamageByEntityEvent $event) : void {
        $this->sendActivation($attacker, "§bParalyze §aActivated!");
        $this->sendActivation($victim, "§cStruck by §bParalyze §cEnchant! Health Effects removed");
        $victim->getEffects()->remove(VanillaEffects::REGENERATION());
        $victim->getEffects()->remove(VanillaEffects::HEALTH_BOOST());
        $effect = new EffectInstance(VanillaEffects::WEAKNESS(), $this->getDuration($enchantmentLevel), $this->getLevel($enchantmentLevel), false);
        $victim->getEffects()->add($effect);
        $effect = new EffectInstance(VanillaEffects::BLINDNESS(), $this->getDuration($enchantmentLevel), $this->getLevel($enchantmentLevel), false);
        $victim->getEffects()->add($effect);
    }

}

File: SkyblockCore/src/main/java/dev/skyblock/enchants/bow/Paralyze.java
package dev.skyblock.enchants.bow;
import dev.skyblock.enchants.BaseBowEnchant;
import org.bukkit.entity.Player;
import org.bukkit.event.entity.EntityDamageByEntityEvent;
public class Paralyze extends BaseBowEnchant {
    public static final int ID = 117;
    public Paralyze() { super(ID, "Paralyze", 5); }
    @Override
    public void onHitPlayer(Player shooter, Player hit, EntityDamageByEntityEvent ev, int level) {

    }
}
